Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -2566,111 +2566,111 @@ def condition(self, fragment, stepList, key, hasHarvest):
         'GT' : 'auto:phase1_2017_design',
         'HLTmenu': '@relval2017',
         'Era' : 'Run2_2017',
-        'BeamSpot': 'GaussSigmaZ4cm',
+        'BeamSpot': 'DBdesign',
         'ScenToRun' : ['GenSim','Digi','RecoFakeHLT','HARVESTFakeHLT'],
     },
     '2018' : {
         'Geom' : 'DB:Extended',
         'GT' : 'auto:phase1_2018_realistic',
         'HLTmenu': '@relval2018',
         'Era' : 'Run2_2018',
-        'BeamSpot': 'Realistic25ns13TeVEarly2018Collision',
+        'BeamSpot': 'DBrealistic',
         'ScenToRun' : ['GenSim','Digi','RecoFakeHLT','HARVESTFakeHLT','ALCA','Nano'],
     },
     '2018Design' : {
         'Geom' : 'DB:Extended',
         'GT' : 'auto:phase1_2018_design',
         'HLTmenu': '@relval2018',
         'Era' : 'Run2_2018',
-        'BeamSpot': 'GaussSigmaZ4cm',
+        'BeamSpot': 'DBdesign',
         'ScenToRun' : ['GenSim','Digi','RecoFakeHLT','HARVESTFakeHLT'],
     },
     '2021' : {
         'Geom' : 'DB:Extended',
         'GT' : 'auto:phase1_2022_realistic',
         'HLTmenu': '@relval2022',
         'Era' : 'Run3',
-        'BeamSpot': 'Realistic25ns13p6TeVEOY2022Collision',
+        'BeamSpot': 'DBrealistic',
         'ScenToRun' : ['GenSim','Digi','RecoNanoFakeHLT','HARVESTNanoFakeHLT','ALCA'],
     },
     '2021Design' : {
         'Geom' : 'DB:Extended',
         'GT' : 'auto:phase1_2022_design',
         'HLTmenu': '@relval2022',
         'Era' : 'Run3',
-        'BeamSpot': 'GaussSigmaZ4cm',
+        'BeamSpot': 'DBdesign',
         'ScenToRun' : ['GenSim','Digi','RecoNanoFakeHLT','HARVESTNanoFakeHLT'],
     },
     '2023' : {
         'Geom' : 'DB:Extended',
         'GT' : 'auto:phase1_2023_realistic',
         'HLTmenu': '@relval2023',
         'Era' : 'Run3_2023',
-        'BeamSpot': 'Realistic25ns13p6TeVEarly2023Collision',
+        'BeamSpot': 'DBrealistic',
         'ScenToRun' : ['GenSim','Digi','RecoNano','HARVESTNano','ALCA'],
     },
     '2024' : {
         'Geom' : 'DB:Extended',
         'GT' : 'auto:phase1_2024_realistic',
         'HLTmenu': '@relval2023',
         'Era' : 'Run3',
-        'BeamSpot': 'Realistic25ns13p6TeVEarly2022Collision',
+        'BeamSpot': 'DBrealistic',
```

Comment:
just pointing out for the record that this is also (currently) mismatched and will be fixed if we use this PR:

```
$ conddb list 133X_mcRun3_2024_realistic_v5 | grep BeamSpot
[2023-11-22 12:47:04,547] INFO: Connecting to pro [frontier://PromptProd/cms_conditions]
BeamSpotObjectsRcd                                      -                                                              BeamSpotObjects_Realistic25ns_13p6TeVCollisions_Early2023_v1_mc                 
BeamSpotOnlineHLTObjectsRcd                             -                                                              BeamSpotOnlineObjects_Realistic25ns_13p6TeVCollisions_Early2023_v1_mc           
BeamSpotOnlineLegacyObjectsRcd                          -                                                              BeamSpotOnlineObjects_Realistic25ns_13p6TeVCollisions_Early2023_v1_mc           
SimBeamSpotObjectsRcd                                   -                                                              SimBeamSpot_Realistic25ns13p6TeVEarly2023CollisionVtxSmearingParameters_v2_mc   
```